src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public String getRandomBall() {
+        Random random = new Random();
+        int qwe = random.nextInt(100);
```

Comment:
The variable name 'qwe' is not informative. Use a meaningful name that describes the variable's purpose, such as 'number'.